mkdocs.yml

```diff
@@ -0,0 +1,62 @@
+site_name: AdaptiveCpp Documentation
+
+repo_url: https://github.com/AdaptiveCpp/AdaptiveCpp
+repo_name: AdaptiveCpp
+
+copyright: Copyright(C) 2021-2023 Timothée David--Cléris
+
+docs_dir: doc
+
+nav:
+  - 'Acpp documentation': 'index.md'
+
+
+  - 'Usage' :
+      - 'Using Acpp' : 'using-hipsycl.md'
+      - 'Compilation' : 'compilation.md'
```

Comment:
```suggestion
      - 'Compilation model' : 'compilation.md'
```

Reply:
This page is perhaps just as much "AdaptiveCpp design" as it is usage. I would suggest to not distinguish between those two categories. In order to explain usage, frequently some design concepts need to be explained as well.